.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
How does `end_of_line` interact with Git for Windows? IIRC, the default behavior is for git to convert all line endings to windows style (`rflf`) on fetch/pull and to convert back on push. Setting the editor to use `lf` line endings is likely to mess with this.

Perhaps the Editorconfig people have worked this out. I don't know, I haven't checked. But I'm not comfortable with this until that is confirmed.

Reply:
I'm not too sure on that - if we're unsure we can simply remove that line.

Reply:
A few of the biggest and related projects' editorconfig files for comparison:
https://github.com/squidfunk/mkdocs-material/blob/master/.editorconfig
https://github.com/h5bp/html5-boilerplate/blob/master/.editorconfig
https://github.com/twbs/bootstrap/blob/v4-dev/.editorconfig
https://github.com/Microsoft/vscode/blob/master/.editorconfig

Reply:
Removed `end_of_line = lf`